### PR TITLE
Add import_rows to Query API

### PR DESCRIPTION
**Pull request by labkey-alan**
#### Rationale
This PR adds import_rows to the Query API. Accessible via api wrapper as api.query.import_rows. This resolves [Issue 50404](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=50404).

Note: while as far as I can tell this does appear to be the correct implementation I cannot seem to get my server to accept a file when using `importLookupByAlternateKey`, it always complains that a string is not an integer, though maybe I am doing something wrong?

#### Related Pull Requests
* n/a

#### Changes
* Add import_rows to Query API
* Improve typing for files

### CODE REVIEW

File: labkey/query.py [hunk 1]
```diff
@@ -357,6 +357,56 @@ def insert_rows(
...
+    :param audit_behavior: Set the level of auditing details for this import action. Available options are "SUMMARY" and
+    "DETAILED". SUMMARY - Audit log reflects that a change was made, but does not mention the nature of the change.
+    DETAILED - Provides full details on what change was made, including values before and after the change. Defaults to
+    the setting as specified by the LabKey query.
+    :param import_lookup_by_alternate_key: Allows lookup target rows to be resolved by values rather than the target's
+    primary key. This option will only be available for lookups that are configured with unique column information
+    :return:
+    """
+    url = server_context.build_url("query", "import.api", container_path=container_path)
+    file_payload = {"file": data_file}
```
Comment (labkey-nicka) [inline labkey/query.py:392]: > Note: while as far as I can tell this does appear to be the correct implementation I cannot seem to get my server to accept a file when using `importLookupByAlternateKey`, it always complains that a string is not an integer, though maybe I am doing something wrong?

I am able to use the `importLookupByAlternateKey` successfully in a JS script with two sample types where one has a lookup to the other. What structures were you using? 

I think an integration test for this wrapper would serve us well.

Comment (labkey-alan) [reply labkey/query.py:392]: I was using a sample with a lookup to a dataclass, and was trying to import via the data class name, which I thought would work. I'll try with a lookup to samples.

Comment (labkey-alan) [reply labkey/query.py:392]: Yeah, that works

**Review by labkey-nicka — APPROVED** [review]
Thanks for adding those tests.

**Pull request merged**